tests/src/main/java/org/hibernate/beanvalidation/tck/tests/xmlconfiguration/ClockProviderSpecifiedInValidationXmlNoDefaultConstructorTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+* JBoss, Home of Professional Open Source
+* Copyright 2009, Red Hat, Inc. and/or its affiliates, and individual contributors
```

Comment:
I'll change it to 2016. I've also filed https://hibernate.atlassian.net/browse/BVTCK-92 to get rid of this lengthy header altogether.